### PR TITLE
Add higher-order reducer for including only specific actions

### PR DESCRIPTION
**Pull request by maxmechanic**
I was curious about what it might look like to have a complementary function to use predicate arrays/functions to exclusively include specific actions, so I sketched this out. Might be useful in terms of expressiveness.

Tests can likely be slimmed/collapsed, but I wanted to pitch the idea.

### CODE REVIEW

**Comment by davidkpiano** [comment]
This was the idea that I had initially - filtering actions rather than filtering out unwanted actions seemed more intuitive to me (see https://github.com/rackt/redux/issues/912).

I can see this being useful, if it were renamed to `filterActions` instead of `includeActions` (which semantically does not preclude unwanted actions, whereas "filtering" does). What do you think, @omnidan ?

**Comment by maxmechanic** [comment]
I think I initially had it as `exclusivelyIncludeActions`, but that felt a little too verbose.
`filterActions` is a good call.

**Comment by omnidan** [comment]
@maxmechanic Thanks! I like it - I was thinking about calling it `onlyActions`, but `filterActions` is probably better.

Also make sure to document the new function in the `README.md` :grin:

**Comment by omnidan** [comment]
@maxmechanic how about we export `ignoreActions` by default? Then this wouldn't be a breaking change and the name is redux-**ignore** after all.

File: README.md [hunk 1]
```diff
@@ -15,9 +15,13 @@ npm install --save redux-ignore
 ## API
 
 ```js
-import ignoreActions from 'redux-ignore';
+import ignoreActions, {filterActions} from 'redux-ignore';
```
Comment (omnidan) [inline README.md]: Maybe we should still encourage using `import { ignoreActions, filterActions }`? It looks cleaner when using both.

**Comment by omnidan** [comment]
@maxmechanic some more comments ^ after these are fixed it should be fine :smiley: Sorry for not explaining it well enough.

BTW, make sure to ping me (@omnidan) when you think it's ready - otherwise I don't get notified about it and it might take forever for me to respond :stuck_out_tongue:

**Comment by maxmechanic** [comment]
@omnidan No worries! I was thinking more about consistency initially, but didn't think about reconciling it with non-breaking changes.

**Comment by omnidan** [comment]
@maxmechanic looks good now! Thanks a lot! :grin:

I wouldn't sacrifice structure for non-breaking-ness (is that a word?), but if it's easy to avoid - why not :smile:

**Comment by omnidan** [comment]
Released 1.2.0 with your changes.

**Pull request merged**